### DISCUSSION TITLE
DVD-SUB and PGSSUB Subtitles only available with transcoding?

### DISCUSSION THREAD

**Discussion by slamdown251** [jellyfin/jellyfin#12402] [Q&A] [answered]
Hi,

I've done some research but it is still not clear to me: Is it the standard Jellyfin behaviour that I can only view subtitles (types dvdsub and pgssub embedded in mkv container encoded with handbrake) when I activate video transcoding so they get burned in? When I disable video transcoding Jellyfin shows no subtitles at all, I've tested it with the web player (Edge), on my TV with the Tizen Jellyfin app installed and on my tablet with the android app. When I play the mkv-files locally with VLC the subtitles are fine.

Thanks for your answers.

**Comment by Shadowghost** [comment, marked as answer]
PGS and DVD-Sub are both picture based subtitles and currently require burning them in if the player is unable to handle them.
10.10 will allow PGS handling in browsers
